README.md

```diff
@@ -67,6 +68,20 @@
 
 - [Signal Processign with GW150914 Open Data](https://losc.ligo.org/s/events/GW150914/GW150914_tutorial.html) - Typical signal processing tasks with the LIGO GW150914 Open Data.
 
+## Optimization
+
+### Learning
+
+- [Convex optimization Solution manual](https://egrcc.github.io/docs/math/cvxbook-solutions.pdf) - Boyd, S.P. and Vandenberghe, L., 2004. Convex optimization. Cambridge university press.
+- [Additional exercises](https://egrcc.github.io/docs/math/cvxbook-solutions.pdf) - Boyd, S.P. and Vandenberghe, L., 2004. Convex optimization. Cambridge university press.
+- [EE364A - Convex Optimization I](https://see.stanford.edu/Course/EE364A) and [Material](https://web.stanford.edu/class/ee364a/) - Stanford Engineering Everywhere - Convex Optimization I course.
```

Comment:
Hi! the Stanford course platform "Stanford Engineering Everywhere" is free (in the "free beer" sense") and open. There is not charge to access this course, at all.